docs/community_resources/faq.md

```diff
@@ -0,0 +1,126 @@
+# FAQ
+
+This page contains answers for most common questions and tips around support and maintenance,
+as well as references to important parts of the documentation and tools useful for developers in their daily work.
+
+#### What information should I specify when creating a Customer Support ticket?
+
+When reporting a problem to Customer Support the most important information is the version of eZ Platform which is used in the project.
+The best way to specify it is to provide the list of currently installed packages by running:
+
+``` bash
+composer show ezsystems/*
+```
+
+Besides that, all the configuration from the `app/config` directory may be helpful.
+
+You should also list the steps to reproduce the issue,
+or at least provide a clear description of the circumstances under which the problem occurred.
+
+If you stumble upon a database-related problem, providing corresponding logs is also an important step.
+
+Additionally, mention recent changes, performed migrations or external scripts/code customizations
+related to the code which generates the problem.
+
+#### What are the recommended ways to increase my project's performance?
+
+The most important clues around increasing overall performance of your eZ Platform-based project can be found in [the Performance documentation page](../guide/performance.md).
+
+#### How can I translate my Back Office?
+
+The language of the Back Office is based on the browser language.
+In order to change it you should install the proper package for your language (see [language packages list](https://github.com/ezplatform-i18n)).
+Once you have language packages installed, you can switch the language of the Back Office in the User Settings menu.
+
+If you do not have a language defined in the browser, it will be selected based on the `parameters.locale_fallback` parameter located in `default_parameters.yml`.
+
+To read more about language managing in eZ Platform, see the following doc pages:
+
+- [Back Office languages](https://doc.ezplatform.com/en/latest/guide/internationalization/#back-office-languages)
+- [Setting up multi-language SiteAccesses and corresponding translations](https://doc.ezplatform.com/en/latest/cookbook/setting_up_multi_language_siteaccesses/)
+
+#### How can I contribute code to eZ Platform?
```

Comment:
section to be changed to be about `How can I apply patches to the installation`
Where the patch can be one given by support, own or community.